### PR TITLE
Templating minter

### PR DESCRIPTION
**Pull request by fadeev**
Using:

```
npx hardhat omnichain Minter recipient:address
```

Related: https://github.com/zeta-chain/docs/pull/91

### CODE REVIEW

File: omnichain/minter/tasks/deploy.ts [hunk 1]
```diff
@@ -14,21 +12,19 @@ const main = async (args: any, hre: HardhatRuntimeEnvironment) => {
...
+  const systemContract = getAddress("systemContract", "zeta_testnet");
 
-  const factory = await hre.ethers.getContractFactory(contractName);
+  const factory = await hre.ethers.getContractFactory("Minter");
   const contract = await factory.deploy(
     "Wrapped tBTC",
     "WTBTC",
-    bitcoinChainID,
-    SYSTEM_CONTRACT
+    18332,
```
Comment (andresaiello) [inline omnichain/minter/tasks/deploy.ts:21]: why did you remove the const? I prefer the const, make it easier to understand

Comment (fadeev) [reply omnichain/minter/tasks/deploy.ts:21]: So that there are fewer changes to the template. I can revert this.

Comment (fadeev) [reply omnichain/minter/tasks/deploy.ts]: f9cd800

**Review by andresaiello — APPROVED** [review]
